app/controllers/autosign_controller.rb

```diff
@@ -3,13 +3,16 @@ def index
     @proxy = SmartProxy.authorized(:view_smart_proxies_autosign).find(params[:smart_proxy_id])
     setup_proxy
 
-    autosign = @api.autosign
-    @autosign = autosign.paginate :page => params[:page], :per_page => Setting::General.entries_per_page
+    @autosign = @api.autosign
```

Comment:
Are you giving up pagination completely? This will definitely break the page. Can you make sure the `Setting::General.entries_per_page` global setting is used? Mabe we should introduce new variable `Setting::General.entries_per_embedded_page` for inlined tables which will perhaps need smaller number than ordinary index pages.